website/docs/r/virtual_machine.html.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "azurerm"
 page_title: "Azure Resource Manager: azurerm_virtual_machine"
-sidebar_current: "docs-azurerm-resource-compute-virtual-machine"
+sidebar_current: "docs-azurerm-resource-compute-virtual-machine-x"
```

Comment:
Was the `-x` supposed to be added?

Reply:
Yep, there's an incorrect highlight on the sidebar at the moment